docs/docsite/rst/dev_guide/developing_plugins.rst

```diff
@@ -473,7 +473,7 @@ Vars plugins
 
 Vars plugins inject additional variable data into Ansible runs that did not come from an inventory source, playbook, or command line. Playbook constructs like 'host_vars' and 'group_vars' work using vars plugins.
 
-Vars plugins were partially implemented in Ansible 2.0 and rewritten to be fully implemented starting with Ansible 2.4. Vars plugins are unsupported by collections.
+Vars plugins were partially implemented in Ansible 2.0 and rewritten to be fully implemented starting with Ansible 2.4. Vars plugins are supported by collections starting with Ansible 2.10.
```

Comment:
You should rather use `version_added` tag here.

Reply:
> You should rather use `version_added` tag here.

I'm not sure what you mean by that. I just corrected the sentence that was there before. How is the correct way to do it?

Reply:
At this point, that rst page doesn't use `version_added` tags at all (for better or worse). I also think we've lost the ability to 'suppress' those tags for very old versions after the split to collections, so I'm inclined to accept this PR as is. If we want to start/go back to using `version_added` we'll need to verify it actually works and doesn't collide with collection versions vs Ansible versions.

Reply:
It would be hard to re-phrase this with the `version_added` tag, since the entire paragraph is a history of vars plugins, and there's no clear/exact version in which vars plugins were added. Although there's an argument to be made for deleting the entire paragraph, I like the current PR, which makes things a little bit clearer and more correct.